code/modules/clothing/suits/armor.dm

```diff
@@ -224,21 +224,19 @@
 
 //All of the armor below is mostly unused
 
-/obj/item/clothing/suit/armor/centcom
-	name = "\improper CentCom armor"
+/obj/item/clothing/head/helmet/space/hardsuit/swat/centcom
+	name = "\improper CentCom SWAT helmet"
+	icon_state = "centcomspace"
+	inhand_icon_state = "centcomspacehelmet"
+	desc = "A tactical MK.II SWAT helmet boasting better protection and a reasonable fashion sense."
+
+/obj/item/clothing/suit/space/hardsuit/swat/centcom
+	name = "\improper CentCom SWAT armor"
 	desc = "A suit that protects against some damage."
```

Comment:
A comically generic description.

Reply:
```suggestion
	desc = "A tactical suit designed for Nanotrasen's most prestigious."
```

Reply:
That was the description of the old Centcom armour, I just didn't change it 'cause it was 7 am and I hadn't slept lol.